.github/workflows/manual-from-secrets.yaml

```
@@ -99,32 +99,12 @@ jobs:
-            exit 1
-          fi
-          echo "::warning::Push attempt $attempt/5 failed (remote is ahead), rebasing and retrying..."
-          # Clean up unstaged changes left by bun install / npm install so that
-          # git pull --rebase can proceed. Do this right before rebase, not before
-          # artifact upload (which happens after push succeeds or fails).
-          git checkout -- .
-          git clean -fd
-          git pull --rebase
-          sleep $((RANDOM % 10 + 5))
-        done
+      uses: ./.github/actions/publish-with-retry
```

actionlint (1.7.12)

[error] could not parse action metadata in "/home/jailuser/git/.github/actions/publish-with-retry": yaml: line 25: could not find expected ':'

(action)